invenio_vocabularies/alembic/4a9a4fd235f8_create_vocabulary_schemes.py

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (C) 2021 Northwestern University.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Create vocabulary schemes."""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '4a9a4fd235f8'
+down_revision = '6312f33645c1'
```

Comment:
comment for reviewers: this point to the affiliations table recipe, so we are all good